NoSoft-SA/nspack
# Phyto Reports

Phyto Reports

Summary of this change:
Added ability for the system to 'remember' phyto_pallet selection i.e. save the selection to db tables.


CHANGELOG entry:

### Added
- phyto_report & phyto_report_pallets tables

How to test this code via the UI:
Finished Goods | Inspection | Phyto Reports
1. Create phyto_report: select pack_tm_group and/or target_market. Then select (stock) pallets to add to the report.
2. View phyto_report
3. Manage Stock Pallets: See a stock_pallets grid with pallets to belong to this phy_to_report already selected. De-select those you want to remove and select those you want to remove.
4. Manage Shipped Pallets: See a shipped_pallets grid with add and remove button on top. User will only be able to see this grid if they have the permission(can_change_inspected_pallets).
5. Delete phyto_report: User will only be able to delete report if they have the permission(can_change_inspected_pallets).
